tools/autograd/derivatives.yaml

```diff
@@ -1450,8 +1450,9 @@
   self: split_with_sizes_backward(grads, split_sizes, dim, self.sizes(), self.options())
   result: auto_linear
 
+# rdar://85629434 - Changed 2 to 2.0 temporarily
```

Comment:
Do we want to keep the rdar reference here?
I think we can associate that with a special issue if you want.

Reply:
Its not needed anymore , we are casting the scalars. This is stale.